### PR TITLE
[filebeat] using https to connect to elasticsearch

### PR DESCRIPTION
**Pull request by framsouza**
This PR will enable filebeat to connect into Elasticsearch via https

### CODE REVIEW

**Comment by framsouza** [comment]
jenkins test this pipeline

**Comment by jmlrt** [comment]
ok to test

**Comment by framsouza** [comment]
jenkins test this pipeline please

File: filebeat/examples/upgrade/Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ include ../../../helpers/examples.mk
 
 CHART := filebeat
 RELEASE := helm-filebeat-upgrade
-FROM := 7.9.0 # registry file version 1 not supported error with previous version
+FROM := 7.15.0 # registry file version 1 not supported error with previous version
```
Comment (jmlrt) [inline filebeat/examples/upgrade/Makefile:7]:
```suggestion
FROM := 7.17.1	# upgrade from versions before 7.17.1 isn't compatible with 8.x
```

File: filebeat/examples/upgrade/test/goss.yaml [hunk 2]
```diff
@@ -25,11 +25,10 @@ user:
     gid: 1000
 
 http:
-  http://upgrade-master:9200/_cat/indices:
+  https://upgrade-master:9200/_cat/indices:
     status: 200
+    allow-insecure: true
     timeout: 2000
-    body:
-      - "filebeat-8.1.0"
```
Comment (jmlrt) [inline filebeat/examples/upgrade/test/goss.yaml:32]: the body should stay, also will require :
```
username: '${ELASTICSEARCH_USERNAME}'
password: '${ELASTICSEARCH_PASSWORD}'
```

**Review by jmlrt — CHANGES_REQUESTED** [review]

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Comment by framsouza** [comment]
ok to test

**Comment by framsouza** [comment]
jenkins test this pipeline please

**Comment by jmlrt** [comment]
ok to test

**Comment by jmlrt** [comment]
ok to test

**Comment by jmlrt** [comment]
ok to test

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Pull request merged**